packages/elements/src/pill/index.ts

```diff
@@ -95,7 +100,37 @@ export class Pill extends ControlElement {
     this.addEventListener('tapstart', this.onStartPress);
     this.addEventListener('tapend', this.onEndPress);
     this.addEventListener('mouseleave', this.onEndPress);
+    this.addEventListener('keydown', this.onKeyDown);
+
+  }
+
+  /**
+   * Updates the element
+   * @param changedProperties Properties that has changed
+   * @returns {void}
+   */
+  protected update (changedProperties: PropertyValues): void {
```

Comment:
Should we use `willUpdate()` or `update()` ?

Reply:
I think we should `update()`